blueprints/aws-alb-crossplane/gatewayclassblueprint-aws-alb-crossplane.yaml

```diff
@@ -337,6 +347,67 @@ spec:
                 tv2.dk/gw: {{ .Gateway.metadata.namespace }}-{{ .Gateway.metadata.name }}
             toPort: 15021
             type: ingress
+      hpa: |
+        {{ if or (get .Values.hpa "minReplicas") (get .Values.hpa "maxReplicas") }}
```

Comment:
What if only `maxReplicas` is specified? What value does `minReplicas` default to?

Reply:
The default comes from the blueprint:

```
apiVersion: gateway.tv2.dk/v1alpha1
kind: GatewayClassBlueprint
...
spec:
  values:
    default:
      ...
      hpa:
        minReplicas: 1
```